lib/offline.js

```diff
@@ -58,14 +59,29 @@ module.exports = promisify(function(config, callback) {
       return '';
     }
   }).then(function(cacheId) {
+    var staticFileGlobs = fileGlobs.map(function(v) {
+      return path.join(rootDir, v)
+    });
+
+    staticFileGlobs.forEach(function(globPattern) {
+      glob.sync(globPattern.replace(path.sep, '/')).forEach(function(file) {
+        var stat = fs.statSync(file);
+
+        if (stat.isFile() && stat.size > 2 * 1024 * 1024 && staticFileGlobs.indexOf(file) === -1) {
+          gutil.log(gutil.colors.red.bold(file + ' is bigger than 2 MiB. Are you sure you want to cache it?'));
```

Comment:
It'd also help to display a message about how to suppress these warnings, if any of them are displayed. It could say something like "To suppress 'file is bigger' warnings, explicitly include the file in the list of files to include." We can do this in a followup, however.